packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -4,19 +4,50 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the background color of the component.
```

Comment:
I think this file should be removed from this PR.

Reply:
Thanks for the catch. 🎣 Fixing!